core/runtime/TRTEngine.cpp

```diff
@@ -318,6 +318,23 @@ void TRTEngine::verify_serialization_fmt(const std::vector<std::string>& seriali
           << ")");
 }
 
+FlattenedState TRTEngine::__obj_flatten__() {
```

Comment:
Im not sure why this needs to be a method vs reusing the functions in register_jit_hooks